src/tabslide.ts

```diff
@@ -42,8 +42,8 @@ export class TabSlide extends TabTie {
     this.setAttribute('type', 'TabSlide');
 
     if (!direction) {
-      const first_fret = ((notes.first_note as unknown) as TabNote).getPositions()[0].fret;
-      const last_fret = ((notes.last_note as unknown) as TabNote).getPositions()[0].fret;
+      const first_fret = (notes.first_note as unknown as TabNote).getPositions()[0].fret;
+      const last_fret = (notes.last_note as unknown as TabNote).getPositions()[0].fret;
```

Comment:
Now that TabNote is migrated I believe you can simply use:
`notes.first_note as TabNote`
and
`notes.last_note as TabNote`